src/activities/menu/Menu.qml

```diff
@@ -77,6 +78,8 @@ ActivityBase {
 
     // @cond INTERNAL_DOCS
     property string url: "qrc:/gcompris/src/activities/menu/resource/"
+    property int  cursorPosition: 0
+    property  string inputText: ""
```

Comment:
on setting inputText to input.text I get this error
"ReferenceError: input is not defined"